hasura/graphql-engine
# forward cookie headers from remote servers (fix #1654)

### Description


Forward only `Set-Cookie` header received from remote schema servers, to client.

**Note**: This won't send back any remote response headers when the websocket transport is used.

### Affected components 


- Server
- Tests

### Related Issues



#1654 
### Solution and Design


Introduced new data type `HttpResponse` which encapsulates headers and some response body.  Changed corresponding handlers. It only sends `Set-Cookie` header from remote.

### Steps to test and verify


1. Add a remote schema which returns say `Set-Cookie` header
2. Make a remote schema query
3. Observe if the cookie is set in the browser

### Limitations, known bugs & workarounds
